### DISCUSSION TITLE
I tried to install Homebrew because I have to install Gnuplot for a university work but I'm ignorant on the topic

### DISCUSSION THREAD

**Discussion by Ncole-png** [Homebrew/discussions#5680] [Getting started] [answered]
### Output of `brew config`

```shell
I don't understand how all this works
```

### Output of `brew doctor`

```shell
It's my first time in the world of programming
```

### Description of issue

==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R nicolecadoria:admin /usr/local/Homebrew
==> Downloading and installing Homebrew...
==> Updating Homebrew...
==> Installation successful!

==> Homebrew has enabled anonymous aggregate formulae and cask analytics.
Read the analytics documentation (and how to opt-out) here:
  https://docs.brew.sh/Analytics
No analytics data has been sent yet (nor will any be during this install run).

==> Homebrew is run entirely by unpaid volunteers. Please consider donating:
  https://github.com/Homebrew/brew#donations

==> Next steps:
- Run these commands in your terminal to add Homebrew to your PATH:
    echo >> /Users/nicolecadoria/.bash_profile
    echo 'eval "$(/usr/local/bin/brew shellenv)"' >> /Users/nicolecadoria/.bash_profile
    eval "$(/usr/local/bin/brew shellenv)"
- Run brew help to get started
- Further documentation:
    https://docs.brew.sh

**(base) nicolecadoria@MacBook-Air-di-Nicole ~ % echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/homefolder/.zprofile
    eval $(/opt/homebrew/bin/brew shellenv)
zsh: no such file or directory: /Users/homefolder/.zprofile
zsh: no such file or directory: /opt/homebrew/bin/brew
(base) nicolecadoria@MacBook-Air-di-Nicole ~ %  echo >> /Users/nicolecadoria/.bash_profile
    echo 'eval "$(/usr/local/bin/brew shellenv)"' >> /Users/nicolecadoria/.bash_profile
zsh: permission denied: /Users/nicolecadoria/.bash_profile
zsh: permission denied: /Users/nicolecadoria/.bash_profile
(base) nicolecadoria@MacBook-Air-di-Nicole ~ %** 

This is the code that appear me after the installation. I cannot run the commands on the terminal because there is "no such file or directory"?
I apologize in advance for my English, but I seriously need help and I don't understand how all this works

**Comment by SMillerDev** [comment]
```
(base) nicolecadoria@MacBook-Air-di-Nicole ~ % echo 'eval $(/opt/homebrew/bin/brew shellenv)' >> /Users/homefolder/.zprofile
eval $(/opt/homebrew/bin/brew shellenv)
```

Why did you run this? The output you pasted tells you to run these 3 commands:
```
echo >> /Users/nicolecadoria/.bash_profile
echo 'eval "$(/usr/local/bin/brew shellenv)"' >> /Users/nicolecadoria/.bash_profile
eval "$(/usr/local/bin/brew shellenv)"
```

**Reply by Ncole-png** [reply]
I tried many times, and the first code doesn't send me anywhere so, I tried different things..
If I run all the three commands, doesn't change anything..

(base) nicolecadoria@MacBook-Air-di-Nicole ~ % 
echo >> /Users/nicolecadoria/.bash_profile
echo 'eval "$(/usr/local/bin/brew shellenv)"' >> /Users/nicolecadoria/.bash_profile
eval "$(/usr/local/bin/brew shellenv)"
zsh: permission denied: /Users/nicolecadoria/.bash_profile
zsh: permission denied: /Users/nicolecadoria/.bash_profile
(base) nicolecadoria@MacBook-Air-di-Nicole ~ %

**Reply by carlocab** [reply]
What's the output of
```
whoami
ls -la /Users/nicolecadoria/.bash_profile
```
?

**Reply by Ncole-png** [reply]
I mean, I'm really ignorant about it ahahah 
I should write "whoami ls -la /Users/nicolecadoria/.bash_profile" and then press send?

Because this is what comes out:

whoami
ls -la /Users/nicolecadoria/.bash_profile
nicolecadoria
-rw-r--r--  1 root  staff  447  6 Ott 23:35 /Users/nicolecadoria/.bash_profile
(base) nicolecadoria@MacBook-Air-di-Nicole ~ %

**Reply by carlocab** [reply]
Do `sudo chown $(whoami) /Users/nicolecadoria/.bash_profile` (this will ask for your password) and then try
```
echo >> /Users/nicolecadoria/.bash_profile
echo 'eval "$(/usr/local/bin/brew shellenv)"' >> /Users/nicolecadoria/.bash_profile
eval "$(/usr/local/bin/brew shellenv)"
```
again.

**Reply by Ncole-png** [reply]
I did it and nothing comes out
is this right?

(base) nicolecadoria@MacBook-Air-di-Nicole ~ % 
sudo chown $(whoami) /Users/nicolecadoria/.bash_profile
Password:
(base) nicolecadoria@MacBook-Air-di-Nicole ~ % 
echo >> /Users/nicolecadoria/.bash_profile
echo 'eval "$(/usr/local/bin/brew shellenv)"' >> /Users/nicolecadoria/.bash_profile
eval "$(/usr/local/bin/brew shellenv)"
(base) nicolecadoria@MacBook-Air-di-Nicole ~ %

**Reply by carlocab** [reply, marked as answer]
Yes that's fine. You should be able to run `brew doctor` now, which should hopefully say `Your system is ready to brew.`. After that, run `brew help` to see help text on how to use `brew`.

**Reply by Ncole-png** [reply]
Thank you so much!
